tests/FakeItEasy.IntegrationTests/ContainerTests.cs

```diff
@@ -1,34 +1,13 @@
 namespace FakeItEasy.IntegrationTests
 {
     using System;
-    using System.Collections.Generic;
-    using Creation;
-    using FakeItEasy.Core;
+    using FluentAssertions;
     using NUnit.Framework;
 
     [TestFixture]
     public class ContainerTests

```

Comment:
Does this fixture still make sense? It only contains a test that covers dummy factories. Perhaps this that test should be moved somewhere else? In any case, it has nothing to do with containers, since containers don't exist anymore.